src/dsnotificationinterface.cpp

```diff
@@ -36,10 +36,16 @@ void CDSNotificationInterface::UpdatedBlockTip(const CBlockIndex *pindexNew, con
     instantsend.UpdatedBlockTip(pindexNew);
     mnpayments.UpdatedBlockTip(pindexNew);
     governance.UpdatedBlockTip(pindexNew);
+
```

Comment:
May be it is not obvious place to make this update and more correct way is to create separate notification interface for that, now we are mixing diffirent logic in one class, but it is not really important.